services/acceptance/services/version.test.js

```diff
@@ -93,13 +93,7 @@ describe('Versions service acceptance tests', () => {
           body: { uuid: this.reg.uuid, manifest: manifest }
         };
         await request(req);
-
-        try {
-          await request(req);
-          expect(false).toBeTruthy();
-        } catch (err) {
-          expect(err.statusCode).toEqual(400);
-        }
+        await request(req);
```

Comment:
This is a bad test, it's not actually testing anything now, you should have an expect for the positive case.

Reply:
Well, it does test that the same call can be done twice without throwing an error like before, isn't it?
Care to clarify what this should 'expect'?

Reply:
Done, care to have another look @mandie722 ?